### PR TITLE
Add missing integrated MPIs

### PR DESCRIPTION
**Pull request by Zaphell**
Noticed these were missing, their presence reflects our 3DS capabilities accurately.

### CODE REVIEW

**Comment by Zaphell** [comment]
This behaves nicely, but I would like the frontend to confirm and test these specs before merging!

File: spec/definitions/Mpi/EMerchantPayMpis/EMerchantPayMpis.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+description: EMerchantPay Mpis
+discriminator: name
+required:
+  - name
+properties:
+  name:
+    allOf:
+      - $ref: "#/definitions/MpiName"
+    enum:
+     - Other
```
Comment (colinbird) [inline spec/definitions/Mpi/EMerchantPayMpis/EMerchantPayMpis.yaml]: is this correct? not EMerchantPayMpi ?

**Comment by Zaphell** [comment]
Waiting on Olivier to test this out properly tomorrow.

**Comment by oliviertr** [comment]
Looks good on our end, just need the Datacash/DataCash casing discrepancy issue to be addressed.

**Pull request merged**